services/state/consensus/topic.proto

```diff
@@ -98,4 +99,16 @@ message Topic {
      * If present, enforces access control for message submission to the topic.
      */
     Key submit_key = 10;
+    /**
+     * If present, enforces access control for update/delete of this topic custom fees.
+     */
+    Key fee_schedule_key = 11;
+    /**
+      * If the transaction contains a signer from this list, no custom fees are applied.
+      */
+    repeated Key free_messages_key_list = 12;
+    /*
+     * The custom fee to be assessed during a message submission to this topic
+     */
+    repeated ConsensusCustomFee custom_fees = 13;
```

Comment:
Text should specify behavior of the field and follow correct formatting:
```suggestion
    /**
     * Access control for update/delete of custom fees.
     * <p>
     * If this field is unset, the current custom fees CANNOT be changed.<br/>
     * If this field is set, that `Key` MUST sign any transaction to update
     * the custom fee schedule for this topic.
     */
    Key fee_schedule_key = 11;

    /**
     * A list of "privileged payer" keys.
     * <p>
     * If a submit transaction is signed by _any_ key from this list,
     * custom fees SHALL NOT be charged for that transaction.<br/>
     * This list SHALL _implicitly_ contain the topic `admin_key`,
     * the topic `submit_key`, and the topic `fee_schedule_key`, if any
     * of those keys is set.
     */
    repeated Key free_messages_key_list = 12;

    /**
     * A set of custom fee definitions.<br/>
     * These are fees to be assessed for each submit to this topic.
     * <p>
     * If this list is empty, the only fees charged for a submit to this
     * topic SHALL be the network and node fees.<br/>
     * If this list is not empty, each fee defined in this set SHALL
     * be evaluated for each message submitted to this topic, and the
     * resultant total assessed fees SHALL be charged.<br/>
     * If this list is not empty, custom fees defined here SHALL be
     * charged _in addition to_ the base network and node fees.
     */
    repeated ConsensusCustomFee custom_fees = 13;
```
Note: I am guessing a bit as to behavior in this suggestion; please correct any errors above.